src/entity/users.rs

```diff
@@ -22,3 +36,14 @@ pub struct Model {
 pub enum Relation {}
 
 impl ActiveModelBehavior for ActiveModel {}
+
+impl ActiveModel {
+    pub fn encrypt(self: &mut Self, password_hash: String) {
+        let salt: String = Alphanumeric.sample_string(&mut rand::thread_rng(), 16);
+        let hash: String = argon2::hash_encoded(password_hash.trim().as_bytes(), 
+            &salt.as_bytes(), &ARGON2_CONFIG).unwrap();  
+
+        self.salt = Set(salt);
+        self.password_hash = Set(hash);
+    }
+}
```

Comment:
Нужна пустая строчка в конце файла
```suggestion
}

```